### PR TITLE
Platform updates

### PR DESCRIPTION
**Pull request by SirSpidey**
- update links
- update anchors
- change bluemix to cloud
- add search metadata

### CODE REVIEW

File: getting-started-credentials.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 copyright:
   years: 2015, 2019
-lastupdated: "2018-12-12"
+lastupdated: "2019-03-08"
+
+keywords: Cloud Foundry, authentication,service credentials
```
Comment (jeffpk62) [inline getting-started-credentials.md]: I don't whether this matters at all, but you have a space after the comma that precedes the text of the second keyword string.  You haven't had such spaces elsewhere.

File: getting-started-sdks.md [hunk 2]
```diff
@@ -40,9 +46,10 @@ The following {{site.data.keyword.ibmwatson}} SDKs are supported by {{site.data.
 * [Swift SDK ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://github.com/watson-developer-cloud/swift-sdk){: new_window}
 * [Unity SDK ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://github.com/watson-developer-cloud/unity-sdk){: new_window}
 
-The [API reference ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://{DomainName}/developer/watson/documentation){: new_window} for each service includes information and examples for many of the SDKs, including Java, Node.js, Python, and Ruby.
+The [API reference ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://{DomainName}/apidocs?category=ai){: new_window} for each service includes information and examples for many of the SDKs, including Java, Node.js, Python, Go, and Ruby.
```
Comment (jeffpk62) [inline getting-started-sdks.md]: Do you want to add Swift the list of SDKs, or are you holding off on that one yet?

File: getting-started-variables.md [hunk 3]
```diff
@@ -2,7 +2,11 @@
 
 copyright:
   years: 2015, 2019
-lastupdated: "2019-02-01"
+lastupdated: "2019-03-08"
+
+keywords: environment variable,service alias, VCAP services
```
Comment (jeffpk62) [inline getting-started-variables.md]: Another space between a comma its trailing keyword string.

File: getting-started-sdks.md (hunk 2 above)
Comment (SirSpidey) [reply getting-started-sdks.md]: nice catch

**Pull request merged**